index.html

```diff
@@ -107,7 +112,7 @@ <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
         </section>
     </section>
     
-    <section class="tittleContainer aboutMe">
+    <section class="tittleContainer aboutMe" id="About">
```

Comment:
- [ ] Great job working on your "About me" section! 💯 However, there are still some things that could be improved in order to match the Figma design! 🎨 Please, fix the background image, the size of the icons, and the color of the title! 🤓 🛠️ 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/199803970-6036d5aa-5ea1-42ec-8b65-f893486328aa.png) | ![imageedit_3_6933405350](https://user-images.githubusercontent.com/77069011/199804446-96a3273c-31c5-404c-a756-bc8738b55c6f.gif) |